polylogue/branch_explorer.py

```
@@ -379,8 +418,132 @@ def branch_diff(conversation: BranchConversationSummary, branch_id: str) -> Opti
+  </body>
+</html>
+"""
+
+
+def _branch_template() -> Environment:
+    env = Environment(
+        loader=BaseLoader(),
+        autoescape=select_autoescape(enabled_extensions=("html",)),
+    )
+    env.filters["href"] = lambda path: quote(path.as_posix(), safe="/:") if path else None
+    env.filters["badge_list"] = lambda badges: "".join(badges)
```

Ruff (0.15.9)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)